tools/pipelines/templates/include-publish-npm-package.yml

```diff
@@ -11,9 +11,22 @@ parameters:
   type: string
 
 stages:
+- stage: publish_npm_internal_test
+  dependsOn: build
+  displayName: Publish Internal Test Packages
+  condition: and(succeeded(), eq(variables['testBuild'], true))
+  jobs:
+  - template: include-publish-npm-package-steps.yml
+    parameters:
+      namespace: ${{ parameters.namespace }}
+      feedName: https://pkgs.dev.azure.com/fluidframework/internal/_packaging/test%40Local/npm/registry/
```

Comment:
Curious: below, on line 34, we have this:
`feedName: https://pkgs.dev.azure.com/fluidframework/internal/_packaging/build/npm/registry/`
Instead of `build%40Local`, we have only used `build`. Is that not the case for `test`, making you use `test%40Local`?

Reply:
I'm not too sure about this. Reading this [doc on feeds](https://docs.microsoft.com/en-us/azure/devops/artifacts/concepts/views?view=azure-devops), I think because the internal build and test feed only has only one view, the "local" view, and that's set as the default, we can omit the specific feed view. I updated the changellist to remove the view for the test feed as well to be consistent. @curtisman, do you have any thoughts on this?

Reply:
Yep. you can skip the feed view.